python/ray/tune/BUILD

```diff
@@ -142,74 +178,348 @@ py_test(
+    name = "lightgbm_example",
+    size = "medium",
+    srcs = ["examples/lightgbm_example.py"],
+    deps = [":tune_lib"],
+    tags = ["exclusive", "example"]
+)
+
+py_test(
+    name = "logging_example",
+    size = "medium",
+    srcs = ["examples/logging_example.py"],
+    deps = [":tune_lib"],
+    tags = ["exclusive", "example"],
+    args = ["--smoke-test"]
+)
+
+py_test(
+    name = "mlflow_example",
+    size = "medium",
+    srcs = ["examples/mlflow_example.py"],
+    deps = [":tune_lib"],
+    tags = ["exclusive", "example"]
+)
+
+py_test(
+    name = "mnist_pytorch",
     size = "small",
-    srcs = ["tests/test_tune_save_restore.py"],
+    srcs = ["examples/mnist_pytorch.py"],
     deps = [":tune_lib"],
-    tags = ["exclusive"],
+    tags = ["exclusive", "example", "pytorch"],
+    args = ["--smoke-test"]
 )
 
 py_test(
-    name = "test_tune_server",
+    name = "mnist_pytorch_lightning",
     size = "medium",
-    srcs = ["tests/test_tune_server.py"],
+    srcs = ["examples/mnist_pytorch_lightning.py"],
     deps = [":tune_lib"],
-    tags = ["exclusive"],
+    tags = ["exclusive", "example", "pytorch"],
+    args = ["--smoke-test"]
+)
+
+py_test(
+    name = "mnist_pytorch_trainable",
+    size = "small",
+    srcs = ["examples/mnist_pytorch_trainable.py"],
+    deps = [":tune_lib"],
+    tags = ["exclusive", "example", "pytorch"],
+    args = ["--smoke-test"]
+)
+
+py_test(
+    name = "nevergrad_example",
+    size = "medium",
+    srcs = ["examples/nevergrad_example.py"],
+    deps = [":tune_lib"],
+    tags = ["exclusive", "example"],
+    args = ["--smoke-test"]
+)
+
+py_test(
+    name = "pbt_convnet_example",
+    size = "medium",
+    srcs = ["examples/pbt_convnet_example.py"],
+    deps = [":tune_lib"],
+    tags = ["exclusive", "example"],
+    args = ["--smoke-test"]
+)
+
+py_test(
+    name = "pbt_example",
+    size = "medium",
+    srcs = ["examples/pbt_example.py"],
+    deps = [":tune_lib"],
+    tags = ["exclusive", "example"],
+    args = ["--smoke-test"]
+)
+
+py_test(
+    name = "pbt_function",
+    size = "medium",
+    srcs = ["examples/pbt_function.py"],
+    deps = [":tune_lib"],
+    tags = ["exclusive", "example"],
+    args = ["--smoke-test"]
+)
+
+py_test(
+    name = "pbt_memnn_example",
+    size = "medium",
+    srcs = ["examples/pbt_memnn_example.py"],
+    deps = [":tune_lib"],
+    tags = ["flaky", "exclusive", "example"],
+    args = ["--smoke-test"]
+)
+
+# Requires GPUs. Add smoke test?
+# py_test(
+#     name = "pbt_ppo_example",
+#     size = "medium",
+#     srcs = ["examples/pbt_ppo_example.py"],
+#     deps = [":tune_lib"],
+#     tags = ["exclusive", "example"],
+#     args = ["--smoke-test"]
+# )
+
+# Requires GPUs. Add smoke test?
+# py_test(
+#     name = "pbt_tune_cifar10_with_keras",
+#     size = "medium",
+#     srcs = ["examples/pbt_tune_cifar10_with_keras.py"],
+#     deps = [":tune_lib"],
+#     tags = ["exclusive", "example"],
+#     args = ["--smoke-test"]
+# )
+
```

Comment:
@richardliaw would be great if you could just quickly confirm that it's either ok to skip these tests or tat we should add a non-GPU smoke test.

Reply:
yeah looks like we didn't run this previously too so looks fine for now.